backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/emu/com/badlogic/gdx/utils/reflect/ClassReflection.java

```diff
@@ -272,9 +272,7 @@ static public boolean isAnnotationPresent (Class c, Class<? extends java.lang.an
 					// append to result set
 					if (!duplicate) {
 						Annotation[] copy = new Annotation[result.length + 1];
-						for (int j = 0; j < result.length; j++) {
-							copy[j] = result[j];
-						}
+						System.arraycopy(result, 0, copy, 0, result.length);
 						copy[result.length] = new Annotation(superAnnotations[i]);

```

Comment:
This could be further improved through `Arrays.copyOf(result, result.length + 1)`, which would make it both faster and more readable. (It seems to be [supported by GWT](http://www.gwtproject.org/doc/latest/RefJreEmulation.html#Package_java_util), if anybody is wondering.)